test/PostgresTests.js

```diff
@@ -0,0 +1,29 @@
+import sqlFormatter from "./../src/sqlFormatter";
+import behavesLikeSqlFormatter from "./behavesLikeSqlFormatter";
+
+describe("StandardSqlFormatter", function() {
```

Comment:
You only added some extra logic to Tokenizer logic that is used by all of the dialects, therefore we can move these tests to behavesLikeSqlFormatter.js.